apps/package.json

```diff
@@ -117,6 +117,7 @@
     "grunt-webpack": "^1.0.11",
     "hammerjs": "^2.0.8",
     "history": "^2.0.1",
+    "html2canvas": "^0.5.0-beta4",
```

Comment:
I had moreso been wondering if we should use an earlier version - i.e. 0.4.1 seems to be their latest non-beta release. I don't have any strong opinions and leave it to your discretion.